File: app/src/main/java/com/ngengs/android/popularmovies/stage1/data/ObjectName.java
package com.ngengs.android.popularmovies.stage1.data;

import com.google.gson.annotations.SerializedName;

/**
 * Created by ngengs on 6/15/2017.
 */

@SuppressWarnings({"WeakerAccess", "unused", "DefaultFileTemplate"})
public class ObjectName {
    private int id;
    private String name;
    @SerializedName("iso_639_1")
    private String iso;

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getIso() {
        return iso;
    }

    public void setIso(String iso) {
        this.iso = iso;
    }
}

File: app/src/main/java/com/ngengs/android/popularmovies/apps/data/ObjectName.kt
package com.ngengs.android.popularmovies.apps.data

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.parcelize.Parcelize

/**
 * Created by rizky.kharisma on 10/11/22.
 * @ngengs
 */
@Parcelize
data class ObjectName(
    var id: Int = 0,
    val name: String = "",
    @SerializedName("iso_639_1")
    val iso: String = "",
) : Parcelable
